docs/gpu.md

```diff
@@ -152,10 +152,32 @@ The only scenario where this would be required is if `$LD_LIBRARY_PATH` is modif
 
 ### Testing the GPU support {: #gpu_cuda_testing }
```

Comment:
Currently, this only treats testing if you can _run_ CUDA-enabled software from EESSI. Maybe we can also include a small instruction for testing if _building new CUDA software_ on top of EESSI works properly. Something like:
```
module load CUDA/<something>
nvcc -o hello_world hello_world.cu hello_world.c (or whatever the correct syntax is here :))
./hello_world
```

Reply:
And mention they should test this for each version of CUDA they installed in `host_injections`

Reply:
Makes sense, but that should be done in a separate PR?

Reply:
If you want, sure. I won't block this one over it :) Although I would consider it to be an integral part of "Testing the GPU support" to be honest :)

Reply:
I don't see it as so integral if we focused on software consumers, it's only integral if you want to do development-type work